src/templates/aids/detail.html

```diff
@@ -46,7 +46,11 @@ <h1> {{ aid.name }} </h1>
     {% regroup aid.categories.all by theme as theme_list %}
     <ul class="aid-categories">
         {% for theme in theme_list %}
+        {% if forloop.counter < 5 %}
```

Comment:
```suggestion
        {% if forloop.counter > 4 %}
```
Pour avoir la même expression que plus bas ?

Reply:
oops j'ai lu trop vite ca ne marche pas mon truc. Mais idéalement avoir le même chiffre en tout cas. Peut-être un <= si ca marche ?

Reply:
Yep avec le <= l'harmonisation est bonne. :)